Zend/zend_gc.c

```diff
@@ -1476,6 +1479,7 @@ ZEND_API int zend_gc_collect_cycles(void)
 			 * short of rerunning full GC tracing. What we do instead is to only run
 			 * destructors at this point, and leave the actual freeing of the objects
```

Comment:
Comment should also be updated to avoid confusion when reading this later. Something along the lines of "only run destructors at this point, then automatically re-run GC at most once after calling the destructors."